packages/frontend-2/components/preview/Image.vue

```diff
@@ -109,7 +111,10 @@ const {
   isLoadingPanorama,
   hasDoneFirstLoad,
   isPanoramaPlaceholder
-} = usePreviewImageBlob(basePreviewUrl, { enabled: isInViewport })
+} = usePreviewImageBlob(basePreviewUrl, {
+  enabled: isInViewport,
+  eagerLoad: props.eagerLoad
```

Comment:
dont care about this not being reactive, we only care about this value on initialization